modules/deploy/pages/deployment-option/cloud/aws-privatelink.adoc

```diff
@@ -0,0 +1,393 @@
+        "allowed_principals": ["<principal_1>","<principal_2>", ...]
+        }
+    }
+}
+EOF`
+
+CLUSTER_ID=`curl -vv -X POST \
+    -H "Content-Type: application/json" \
+    -H "Authorization: Bearer $AUTH_TOKEN" \
+    -d "$CLUSTER_POST_BODY" $PUBLIC_API_ENDPOINT/v1beta1/clusters | jq .metadata.cluster_id | sed 's/"//g'`
+
+echo $CLUSTER_ID
+----
++
+**BYOC clusters:** Check that the cluster operation is completed by calling https://docs.api.redpanda.com/#get-/v1beta1/operations/-id-[`GET /v1beta1/operations/{id}`], and passing the operation ID returned from the Create Cluster call.
++
+When the Create Cluster operation is completed (`STATE_COMPLETED`), execute the following `rpk cloud` command to finish creating your BYOC cluster:
++
+[,bash]
+----
+rpk cloud byoc aws apply --redpanda-id=$CLUSTER_ID
+----
+
+== Enable PrivateLink endpoint service for existing clusters
+
+. In the Redpanda Cloud UI, go to the cluster overview and copy the cluster ID from the **Details** section.
++
+[,bash]
+----
+CLUSTER_ID=<cluster_id>
+----
+
+. Make a https://docs.api.redpanda.com/#patch-/v1beta1/clusters/-cluster.id-[`PATCH /v1beta1/clusters/{cluster.id}`] request to update the cluster with the Redpanda Private Link Endpoint Service enabled.
++
+In the example below, make sure to set your own value for the following field:
++
+--
+- `allowed_principals`: Amazon Resource Names (ARNs) for the AWS principals allowed to access the endpoint service. For example, for all principals in an account, use `arn:aws:iam::account_id:root`. See https://docs.aws.amazon.com/vpc/latest/privatelink/configure-endpoint-service.html#add-remove-permission[Configure an endpoint service] for details.
+--
++
+[,bash]
+----
+CLUSTER_PATCH_BODY=`cat << EOF
+{
+  "private_link": {
+     "enabled": true,
+     "aws": {
+        "allowed_principals": ["<principal_1>","<principal_2>", ...]
+     }
+  }
+}
+EOF`
+
+curl -vv -X PATCH \
+  -H "Content-Type: application/json" \
+  -H "Authorization: Bearer $AUTH_TOKEN" \
+  -d "$CLUSTER_PATCH_BODY" $PUBLIC_API_ENDPOINT/v1beta1/clusters/$CLUSTER_ID
+----
+
+. Before proceeding, check the state of the Update Cluster operation by calling https://docs.api.redpanda.com/#get-/v1beta1/operations/-id-[`GET /v1beta1/operations/{id}`], and passing the operation ID returned from Update Cluster call. When the state is `STATE_READY`, proceed to the next step.
+
+. Check the service state by calling https://docs.api.redpanda.com/#get-/v1beta1/clusters/-id-[`GET /v1beta1/clusters/{id}`]. The `service_state` in the `private_link.status` response object must be `Available` for you to <<connect-to-privatelink-endpoint-service,connect to the service>>.
++
+[,bash]
+----
+curl -X GET \
+    -H "Content-Type: application/json" \
+    -H "Authorization: Bearer $AUTH_TOKEN" \
+    $PUBLIC_API_ENDPOINT/v1beta1/clusters/$CLUSTER_ID | jq '.private_link.status.aws | {service_name, service_state}'
+----
+
+== Connect to PrivateLink endpoint service
```

Comment:
Not generally a fan of a heading followed by a heading with no text in between. Is there a meaningful/helpful introduction that makes sense here?